pkg/resolver/mocks/mock-rel.json

```diff
@@ -0,0 +1,41 @@
+
+{
```

Comment:
This look different from `mock-rel-1.json`.  Do you intend to use the same format as `mock-rel-1.json` here?